main/guides/getting-started/swapaoo-making-a-payment-explainer.md

```diff
@@ -0,0 +1,41 @@
+# Making Payments in Agoric Smart Contracts
```

Comment:
I just noticed: the filename has a typo in it. swapaoo should be swaparoo

Reply:
fixed typo